src/styles/blocks.scss/cell.scss

```diff
@@ -0,0 +1,65 @@
+.wrapper {
+  padding-top: 100px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  width: getWidth($cell-size);
+  padding: 20px $calendar-paddings;
+  gap: $cell-gap;
+
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  text-align: center;
+
+  $days:
+    "mon" 0,
+    "tue" 101px,
```

Comment:
what happened if I wanna change cell size? do I need to change all those values?